E2E-tests/src/sidechain_main_cli.py

```diff
@@ -81,6 +81,42 @@ def get_signatures(
             raise e
         return signatures
 
```

Comment:
We need to rename this file too btw

Reply:
actually @chrispalaskas can I do this in a separate PR - changing this file is a big change, because I have to change class names, and error classes all through out our test files.